### PR TITLE
Add rich-text support for feedback session description #5682

### LINKED ISSUE #5682: Add rich-text support for feedback session description
$subj

As rich-text is based on HTML, we need to disable sanitizing for this field. Is it OK to push such initial version to the master until the new module for preventing XSS, etc. is implemented ? Or it's worth additional branch ?

### PR DESCRIPTION
**Pull request by belyabl9**
Fixes #5682

### CODE REVIEW

**Comment by belyabl9** [comment]
Ready to review

File: src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html [hunk 1]
```diff
@@ -203,9 +203,13 @@ <h5 class="col-sm-2 margin-top-0">
...
-                  <textarea class="form-control" cols="100%" disabled="" id="instructions" name="instructions" placeholder="e.g. Please answer all the given questions." rows="4">
-                    Instructions for first session
-                  </textarea>
+                  <div class="panel panel-default panel-body mce-content-body" id="instructions" spellcheck="false">
+                    <p>
+                      Instructions for first session
+                    </p>
+                  </div>
+                  <input name="instructions" type="hidden">
+                  <input name="instructions" type="hidden">
```
Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html]: Any idea what causes this?

Comment (belyabl9) [inline src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html]: When tinyMCE works in an inline mode, it sends value via hidden input. As I remember, duplicate appears after reinitilizing that is done when we choose edit/readonly mode while editing feedback session. I didn't notice any side-effects of duplicate inputs while testing

Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html]: Noted, we can investigate later on if this causes problems in the future.

**Comment by wkurniawan07** [comment]
Looks good. @damithc do we want to merge the rich-text additions all at once or feature by feature? If we opt for the former, this PR should go on a long-lived branch.

**Comment by damithc** [comment]
If this is releasable, I don't mind releasing it. What do you think? @wkurniawan07 @belyabl9 ? If you want to wait, I'm ok with that too. BTW, can you post screenshots of edit screen and displays of a rich text session description (in all places where it is displayed)?

**Comment by belyabl9** [comment]
I think it's releasable

image

image

**Comment by wkurniawan07** [comment]
Ok, let's give this one a release; it'd be good to give a first try as well as some more testing as well. @damithc for your review.

**Comment by wkurniawan07** [comment]
@belyabl9 can you merge master to this branch or rebase it so I can merge this?

**Pull request merged**